sql/telemetry_derived/error_aggregates/query.sql

```diff
@@ -0,0 +1,185 @@
+CREATE TEMP FUNCTION
+  udf_json_extract_int_map (input STRING) AS (ARRAY(
+    SELECT
+      STRUCT(CAST(SPLIT(entry, ':')[OFFSET(0)] AS INT64) AS key,
+             CAST(SPLIT(entry, ':')[OFFSET(1)] AS INT64) AS value)
+    FROM
+      UNNEST(SPLIT(REPLACE(TRIM(input, '{}'), '"', ''), ',')) AS entry
+    WHERE
+      LENGTH(entry) > 0 ));
+CREATE TEMP FUNCTION
+  udf_json_extract_histogram (input STRING) AS (STRUCT(
```

Comment:
Is this function not already available for use? https://docs.telemetry.mozilla.org/cookbooks/bigquery.html#accessing-histograms

Reply:
Because persistent UDFs are still technically in beta, we use temporary UDFs (which are GA) in these ETL queries. You can reference names like `udf_json_extract_histogram` in your queries in the `templates` folder and the  bigquery-etl machinery will inject the temporary UDF into the generated files under `sql`. So this looks fine to me.

I bet persistent UDFs will be GA soon and we'll be able to get rid of the generation machinery altogether.